eth/backend_apis_mobile.go

```diff
@@ -0,0 +1,9 @@
+// +build mobile
+
+package eth
+
+import "github.com/ethereum/go-ethereum/rpc"
+
+func (s *Ethereum) APIs() []rpc.API {
+	return []rpc.API{}
```

Comment:
don't we want the eth_ apis?

Reply:
I believe no api's are being used from the mobile client, but I'll double check